Tools/Frame_params/TradHeli_Copter36_Upgrade-QGC.params

```diff
@@ -12,24 +12,24 @@
 1	1	LOIT_SPEED	3000	9
 1	1	PHLD_BRAKE_ANGLE	800	4
 1	1	PHLD_BRAKE_RATE	4	4
-1	1	PSC_ACCZ_D	0	9
-1	1	PSC_ACCZ_FF	0	9
-1	1	PSC_ACCZ_FILT	20	9
-1	1	PSC_ACCZ_FLTD	20	9
-1	1	PSC_ACCZ_I	1	9
-1	1	PSC_ACCZ_IMAX	800	9
-1	1	PSC_ACCZ_P	0.28	9
+1	1	PSC_ACCD_D	0	9
```

Comment:
my guess is that we should just delete these "TradHeli_xxx_Upgrade_xx" files.  @bnsgeyer what do you think?  If we do delete them we can do it as a separate PR